### DISCUSSION TITLE
Ambient display

### DISCUSSION THREAD

**Discussion by pservais** [Madelena/hass-config-public#19] [Q&A] [answered]
Hi Madalena!

First I would like to thank you for sharing your maximalist dashboard, I've spend quite a few hours integrating it in our home.
My wife isn't quite happy with the amount of info presented, LOL! But when she saw your presentation of the smart display dashboard (release party HA), she was stunned.
Are you planning to release the code for the smart displays on GitHub? I would love to integrate this aa our main dash in the living room.

Greetings from The Netherlands :)

**Comment by jsenecal** [comment, marked as answer]
I believe everything was added in commit a4a8f18755e5f7a27d5564bb11db494cac1fdc35

You can see what @Madelena has done over there :
https://github.com/Madelena/hass-config-public/tree/main/ui/smartdisplays

**Reply by pservais** [reply]
Yep! Big thank you @Madelena !
